### DISCUSSION TITLE
Google: How to get the application version?

### DISCUSSION THREAD

**Discussion by ItsIgnacioPortal** [matamorphosis/Scrummage#48] [Q&A] [answered]
I've created a Programmable Search engine, so I have an API key, the CX parameter, and the application name. But where are you supposed to specify the application version on the google dashboard? It doesn't seem to be an option

**Comment by matamorphosis** [comment, marked as answer]
Hi @PinkDev1 not sure if your question relates to Google or Scrummage. The following screenshot shows how you can enter the details in Scrummage, firstly go to this endpoint and select "google" from the dropdown list:
![image](https://user-images.githubusercontent.com/26025152/150444305-17c7b561-a394-4a29-b2d8-15586df77b39.png)
Click Continue, then you can enter your details:
![image](https://user-images.githubusercontent.com/26025152/150444247-2e63447a-17c2-4521-98e5-224a736a55c6.png)

**Reply by ItsIgnacioPortal** [reply]
The question is about google. Where can I get the `application_version`?

**Reply by matamorphosis** [reply]
@PinkDev1 reading the documentation [here](https://developers.google.com/custom-search/v1/using_rest), you can see in the URL both in your URL bar and on the page a reference to v1. So it's actually related to the programmable search API provided by Google and not your custom search.

Try entering a value of `v1` and let me know how you go.
